### PR TITLE
fix(Synthetics): Removing stale transaction traces doc

### PR DESCRIPTION
**Pull request by jeff-colucci**
Removed deprecated feature, changed the yaml, and added redirects

### CODE REVIEW

File: src/content/docs/synthetics/synthetic-monitoring/using-monitors/view-simple-scripted-monitor-results.mdx [hunk 1]
```diff
@@ -9,6 +9,11 @@ redirects:
   - /docs/new-relic-only/new-relic-synthetics/using-monitors/viewing-monitor-results
   - /docs/synthetics/new-relic-synthetics/using-monitors/viewing-monitor-results
   - /docs/synthetics/new-relic-synthetics/using-monitors/view-monitor-results
+  - /docs/synthetics/synthetic-monitoring/using-monitors/view-simple-scripted-monitor-results
+  - /docs/new-relic-only/new-relic-synthetics/using-monitors/collecting-synthetic-transaction-traces
+  - /docs/synthetics/new-relic-synthetics/using-monitors/collecting-synthetic-transaction-traces
+  - /docs/synthetics/new-relic-synthetics/using-monitors/collect-synthetic-transaction-traces
```
Comment (akristen) [inline src/content/docs/synthetics/synthetic-monitoring/using-monitors/view-simple-scripted-monitor-results.mdx:15]:
```suggestion
  - /docs/synthetics/synthetic-monitoring/using-monitors/collect-synthetic-transaction-traces/
```

I think this is the link that needs to be there. I'm not sure what the other one leads to.

**Review by akristen — CHANGES_REQUESTED** [review]

File: src/content/docs/synthetics/synthetic-monitoring/using-monitors/view-simple-scripted-monitor-results.mdx (hunk 1 above)
Comment (akristen) [reply src/content/docs/synthetics/synthetic-monitoring/using-monitors/view-simple-scripted-monitor-results.mdx:15]: Or maybe this in addition to the other links. In any case, this link needs to be the one to appear on this doc to redirect

Comment (jeff-colucci) [reply src/content/docs/synthetics/synthetic-monitoring/using-monitors/view-simple-scripted-monitor-results.mdx:15]: It looks like both links currently work so I think we can put both in pretty safely!

**Review by akristen — APPROVED** [review]
The redirect worked!

**Pull request merged**